core/backend/src/Model.ts

```diff
@@ -215,6 +215,13 @@ export class Model extends Entity {
   public update() { this.iModel.models.updateModel(this.toJSON()); }
   /** Delete this Model from the iModel. */
   public delete() { this.iModel.models.deleteModel(this.id); }
+
+  // NOTE: non-element entities do not yet have a concept of required references
+  protected override collectReferenceConcreteIds(referenceIds: EntityReferenceSet): void {
+    super.collectReferenceConcreteIds(referenceIds);
+    if (this.parentModel) referenceIds.addModel(this.parentModel);
```

Comment:
Here and elsewhere please follow coding guidelines: body of conditional goes on separate line.
(I wish we were automating enforcement of those guidelines...)

Reply:
I grepped through all `collectReferenceConcreteIds` overrides since I'm touching them and made them follow that guide line. We really need a properly configured lint rule so we can stop caring about it in PR.